.env

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_SUPABASE_URL="https://rmyvutrqvavmemptyxwb.supabase.co"
+NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InJteXZ1dHJxdmF2bWVtcHR5eHdiIiwicm9sZSI6ImFub24iLCJpYXQiOjE3Mjg0MzI4NTYsImV4cCI6MjA0NDAwODg1Nn0.gNMrbihw3QfPfQxgrziUE9Vx0LXmMUqqP3QgvbPc1Tw
```

Comment:
Oops! You should rotate your `anon` key, as it's not safe to publish your environment variables. In Next.js, local environment variables are stored in the `.env.local` file.

Reply:
`.env` file shouldn't be part of the commit